README.md

```diff
@@ -57,6 +60,19 @@ To get started with the latest testnet, see the [docs](https://docs.cronos.org/g
 
 [testnet.json](./testnet.json) lists all the existing testnets information.
 
+## Gravity Bridge Testnet Status
+- _Aug 26, 2022_ - `pioneereleventestnet_340-1`
+  - with `cronosd` [v0.8.0-gravity-alpha1-patch](https://github.com/crypto-org-chain/cronos/releases/tag/v0.8.0-gravity-alpha1-patch)
+
+- _Aug 25, 2022_ - `pioneereleventestnet_340-1`
```

Comment:
maybe we can remove this? `alpha2` is not needed anymore

Reply:
Removed in the last commit  👍